scripts/fast-reboot

```diff
@@ -687,6 +698,10 @@ if [ -x ${LOG_SSD_HEALTH} ]; then
     ${LOG_SSD_HEALTH}
 fi
 
+if [[ ( "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ) && "$TEAMD_INCREASE_RETRY_COUNT" -eq 1 ]]; then
```

Comment:
I suggest to use syntax of `${REBOOT_TYPE}` over `$REBOOT_TYPE`. The former is more safer and predictable.

Reply:
Changed.